L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -207,8 +237,16 @@ std::vector<std::pair<GlobalPoint, double>> HGCalHistoSeedingImpl::computeInterp
         if (isMax) {
           float ROverZ_seed = kROverZMin_ + (bin_R + 0.5) * (kROverZMax_ - kROverZMin_) / nBinsRHisto_;
```

Comment:
There is a lot of duplicated code in all the various seeding methods. It could be put in a separate function.